redash/tasks/queries/maintenance.py

```diff
@@ -131,10 +131,17 @@ def cleanup_query_results():
         settings.QUERY_RESULTS_CLEANUP_MAX_AGE,
     )
 
-    unused_query_results = models.QueryResult.unused(settings.QUERY_RESULTS_CLEANUP_MAX_AGE)
-    deleted_count = models.QueryResult.query.filter(
-        models.QueryResult.id.in_(unused_query_results.limit(settings.QUERY_RESULTS_CLEANUP_COUNT).subquery())
-    ).delete(synchronize_session=False)
+    unused_query_results = models.QueryResult.unused(days=settings.QUERY_RESULTS_CLEANUP_MAX_AGE)
+    deleted_count = len(
```

Comment:
We can use `result.rowcount` instead of calling `len`.

Reply:
updated

Reply:
Thanks. Confirmed.